src/core/config/netdata_publisher.h

```diff
@@ -141,7 +141,7 @@
  *
  */
 #ifndef OPENTHREAD_CONFIG_NETDATA_PUBLISHER_DESIRED_NUM_EXTERNAL_ROUTE_ENTRIES
-#define OPENTHREAD_CONFIG_NETDATA_PUBLISHER_DESIRED_NUM_EXTERNAL_ROUTE_ENTRIES 10
+#define OPENTHREAD_CONFIG_NETDATA_PUBLISHER_DESIRED_NUM_EXTERNAL_ROUTE_ENTRIES 3
```

Comment:
I think we should keep this as 10. I think this value is already specified in the Thread 1.3 spec (section ~9.12)

In general, with external routes, we want more BRs to publish it in netdata which then helps with flow of traffic (provide more chances for a node within Thread mesh to get to a closer BR that route the traffic outside).

I would suggest instead to change/require NAT64 prefix to follow the same rule as other ext routes.

Reply:
agree that they should follow the same rule.